DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/data/DSACampaign.java

```diff
@@ -49,9 +49,9 @@ public DSACampaign(int DSACampaignId, int userId, int keywordCampaignId, String
     this.clicks = clicks;
   }
 
-  public DSACampaign(int DSACampaignId, int userId, int keywordCampaignId, String name, String fromDate, String toDate, double dailyBudget, String location, String domain, String target) {
+  public DSACampaign(int DSACampaignId, String userId, int keywordCampaignId, String name, String fromDate, String toDate, double dailyBudget, String location, String domain, String target) {
```

Comment:
This will break my code, and I can easily do this from my end.

Reply:
Resolved, moved back to ur original code.